python/dasclient/dasclient.py

```diff
@@ -386,16 +390,24 @@ def get_file(self, url):
     def get_event_types(self):
         return self._get('activity/events/eventtypes')
 
+    def get_event_schema(self, event_type):
+        return self._get(f'activity/events/schema/eventtype/{event_type}')
+
     def get_events(self, **kwargs):
         params = dict((k, v) for k, v in kwargs.items() if k in
-            ('state', 'page_size', 'page', 'event_type', 'filter', 'include_notes', 'include_related_events','include_files', 'include_details', 'include_updates', 'max_results'))
+            ('state', 'page_size', 'page', 'event_type', 'filter', 'include_notes',
+            'include_related_events','include_files', 'include_details',
+            'include_updates', 'max_results', 'oldest_update_date'))
         self.logger.debug('Getting events: ', params)
         events = self._get('activity/events', params=params)
 
         count = 0
         while True:
             if events and events.get('results'):
                 for result in events['results']:
+                    if('oldest_update_date' in params):
```

Comment:
sorry if I led you astray on the parens usage. Don't need them here since we aren't breaking the line for code readability.